site/content/installation/installing-ngf/manifests-openshift.md

```diff
@@ -0,0 +1,232 @@
+---
```

Comment:
This whole file is basically the same as the regular installation through manifests, but with just about every installation command modified. I thought this would be a better approach than weaving openshift commands through the regular installation through manifests file. 

Also since installation through Helm is just a single flag, I thought that was best to keep in the regular installation through Helm file.

Reply:
the amount of duplication is huge between this file and  https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/installation/installing-ngf/manifests.md installations

@ADubhlaoich 

I wonder if anything can be done so that we reduce that amount. Some conditional (templated) content-generation, use of includes or something?

Reply:
Removed this file, resolving conflict.